app/screens/login_options/login_options.js

```diff
@@ -209,6 +262,37 @@ export default class LoginOptions extends PureComponent {
         return null;
     };
 
+    renderOpenIdOption = () => {
+        const {config, license} = this.props;
+        const openIdEnabled = config.EnableSignUpWithOpenId === 'true' && license.IsLicensed === 'true' && isMinimumServerVersion(config.Version, 5, 32, 0);
+
+        if (openIdEnabled) {
+            const additionalButtonStyle = {
+                backgroundColor: config.OpenIdButtonColor || '#145DBF',
+                borderColor: 'transparent',
+                borderWidth: 0,
+            };
+
+            const textColor = 'white';
+
+            return (
+                <Button
+                    key='openId'
+                    onPress={preventDoubleTap(() => this.goToSSO(ViewTypes.OPENID))}
+                    containerStyle={[GlobalStyles.signupButton, additionalButtonStyle]}
+                >
+                    <FormattedText
+                        id='signup.openid'
+                        defaultMessage={config.OpenIdButtonText || 'OpenId'}
```

Comment:
Do we want `OpenId` or `OpenID` ? 
Looking at the official logo, I think we want to go with second option.
![image](https://user-images.githubusercontent.com/17804942/103685447-3c774600-4f5b-11eb-8006-78482e4fba43.png)

Reply:
Agree on the second option.